ims/common/config.py

```diff
@@ -73,5 +114,6 @@ def section(self, section_name, required=True):
 
 
 class ConfigSection:
+    """ A Simple Object to be used to get cfg.section.option """
     def __init__(self):
```

Comment:
I needed an object where I can set attributes to. like cfg.iscsi.ip, here iscsi is a ConfigSection obj.